sunpy/tests/test_models.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+
+import sunpy.sun.models as sun_models
+
+
+@pytest.mark.parametrize("model_name", sun_models._MODELS.keys())
+def test_valid_model_loading(model_name):
+    """
+    Test if models in sun_models._MODELS load correctly.
+    """
+    model_data = getattr(sun_models, model_name)
+    assert model_data is not None, f"Test Failed: {model_name} does not exist."
+
+
+@pytest.mark.parametrize("model_name", sun_models._MODELS.keys())
```

Comment:
This can be done for one model, since it should work for one, it will work for all.